app/Controllers/FrontendController.php

```
@@ -1934,6 +1936,15 @@ private function getRelatedBooks(mysqli $db, int $book_id, array $book, array $a
     {
         $related_books = [];
         $limit = 3;
+        $allCreatorsSelect = "
+            (SELECT GROUP_CONCAT(DISTINCT " . \App\Support\AuthorName::displaySql('a_all') . "
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\AuthorName' in method 'getRelatedBooks'. (undefined)

(StaticAccess)